### ISSUE TITLE
search box is coinciding with the chapter list when we scroll down the chapter list

### ISSUE THREAD

**Issue by Neerajkaushik439** [CircuitVerse/CircuitVerseDocs#384]
when we go to documentation section , there is a list of chapters for navigation and there a is a search box , when we scroll down the list , the search  box is coinciding with the list of chapters.

Go to - https://docs.circuitverse.org/#/
         OR
1. Go to the "Getting Started" in the navigation bar then head to the documentation section
2. Scroll down the list of chapters

 Expected behavior :
The list of chapters should not merge with the Search box , there should be a dark background of search section so that chapter list in not visible behind search box

Screenshot :
![image](https://github.com/user-attachments/assets/c67b7439-e41d-497b-b7cc-a95a62d6690b)

 Device Information :
 - OS: Windows
 - Browser: Chrome

I am working on this issue

**Comment by webermayank** [comment]
yes the issue is still there, i would like to work on this issue, may i ?

**Comment by Kaushik-Shahare** [comment]
Can i work on this issue?

**Comment by Me-Priyank** [comment]
UI had been changed
